tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1233,7 +1234,7 @@ def initActivationFunctionObjects(self):
       state["ProblemType"]["GroupedGemm"] = False
       state["KernelLanguage"] = "Source"
       state["Kernel"] = {"WavefrontSize": self["WavefrontSize"], "ISA": tuple(self["ISA"])}
-      self.activationFunctionObjects.append(KernelWriterActivationFunction(state))
+      self.activationFunctionObjects.append(KernelWriterActivationFunction(state, self.cxxCompiler))
```

Comment:
ouch this is why we need to move compile commands out of the Kernel Helpers

Reply:
Spot on!